ethers-solc/src/artifacts/mod.rs

```diff
@@ -1779,6 +1779,7 @@ pub struct Creation {
 
 #[derive(Clone, Debug, Serialize, Deserialize, Eq, PartialEq)]
 pub struct Ewasm {
+    #[serde(default)]
     pub wast: String,
```

Comment:
this is no longer symmetrical.

```suggestion
    #[serde(default, skip_serializing_if = "Option::is_none")]
    pub wast: Option<String>,
```

Reply:
Changing it to `Option<String>` will introduce breaking changes. Is that okay?

Reply:
done

Reply:
is fine imo, almost certain that nobody's using this